clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -213,9 +211,9 @@ func (pc *Controller) SetUpPrometheusStack() error {
 		return err
 	}
 	if pc.clusterLoaderConfig.PrometheusConfig.ScrapeNodeExporter {
-		if err := pc.runNodeExporter(); err != nil {
```

Comment:
I think the reason why I implemented this in such weird way (ssh to nodes) is as the daemonsets approach doesn't work in kubemark. https://github.com/kubernetes/perf-tests/pull/596#issuecomment-510500749

Reply:
I am not familiar with kubemark.  Why do the control planes in kubemark have the `NetworkUnavaiable` condition?

Reply:
Kubemark has incorrect network setup and we never had a time to cleanup this. While that time, the kubemark was quite important to us, now the focus has changed and we rarely use it.

I'm fine with any implementation that won't be scraping on masters on kubemark clusters.